weave/trace_server/opentelemetry/python_spans.py

```diff
@@ -352,7 +355,16 @@ def to_call(
         else:
             turn_id = None
 
-        wb_run_id = wandb_attributes.get("wb_run_id") or None
+        wb_run_id = wb_run_id or wandb_attributes.get("wb_run_id") or None
+
+        if wb_run_id:
+            if wb_run_id.find(":") == -1:
```

Comment:
The condition where ":" is not in the wb_run_id implies that it was not correctly converted on the way in. Can you share why you decided to add this condition? As is, it seems like it might be hiding a bug